submissions/NadiaVorontsova/hooli-style-popup/style.css

```
@@ -0,0 +1,144 @@
+*{
```

please, use space after each css-selector
```suggestion
* {
```

---

I fixed it